package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "app",
+  "private": "true",
+  "scripts": {
+    "build": "webpack --config webpack.config.js",
+    "stylefix": "npx stylelint \"**/*.{css,scss}\" --fix"
+  },
+  "dependencies": {
+    "@babel/core": "^7.23.5",
+    "@babel/preset-env": "^7.23.5",
+    "@babel/preset-react": "^7.23.3",
+    "@reduxjs/toolkit": "^2.0.1",
+    "axios": "^1.6.2",
+    "babel-loader": "^9.1.3",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-redux": "^9.0.1",
+    "react-router": "^6.20.1",
+    "react-router-dom": "^6.20.1"
+  },
+  "devDependencies": {
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0",
+    "webpack-cli": "^5.1.4"
+  }
+}
```

Comment:
- [ ] While your project code appears well written as required because the thing left is to verify if it actually works, there is a configuration issue I run into even when I precompile 
![image](https://github.com/Movgto/react-on-rails-practice/assets/86200356/ecb8beda-98b5-47a6-afa1-596d9e43be2c)
The above is the error log shown in the terminal when I precompile. I would advise you to follow this [tutorial](https://betterprogramming.pub/react-with-rails-2022-bd28e1fcd355#ad0a:~:text=com/es6%2Dmodule-,Webpack,-At%20its%20core) step by step to properly set up this project. 👍 😎

Reply:
I already ran the application on a local server and it was working for me 😕